packages/angular/ssr/src/routes/ng-routes.ts

```diff
@@ -458,6 +458,13 @@ function buildServerConfigRouteTree({ routes, appShellRoute }: ServerRoutesConfi
       continue;
     }
 
+    if (path.includes('**') && 'getPrerenderParams' in metadata) {
+      errors.push(
+        `Invalid '${path}' route configuration: 'getPrerenderParams' cannot be used with a '**' route.`,
```

Comment:
Question: Is this true for a single `*` too? Should we ban that as well?

Reply:
Updated